### DISCUSSION TITLE
How do I register a command with parameters?

### DISCUSSION THREAD

**Discussion by sgtester01** [Flow-Launcher/Flow.Launcher#2297] [Q&A] [answered]
Can I register customized commands?
I want to register a command that will execute the application with parameters.

For example, I want to register "notepad.exe C:\memo.txt" as a "Memo" command, or "python -V" as a "Pyver" command.

**Comment by jjw24** [comment, marked as answer]
Hi there, you should be able to use the Runner plugin for this.

**Reply by sgtester01** [reply]
Thank you! I was able to execute the command with parameters.
